dev/src/recursive-delete.ts

```diff
@@ -84,6 +84,21 @@ export class RecursiveDelete {
    */
   private documentsPending = true;
 
+  /**
+   * Whether run() has been called.
+   * @private
+   */
+  private started = false;
+
+  /** Query limit to use when fetching all descendants. */
+  private maxPendingOps: number;
```

Comment:
Does this need to be marked "private" (here and below)?

Reply:
Thanks for catching, done.